app/assets/javascripts/component_guide/application.js

```diff
@@ -1,3 +1,4 @@
-//= require ./no_slimmer
+//= require govuk_publishing_components/dependencies
+//= require govuk_publishing_components/all_components
```

Comment:
I don't think these includes are needed as they're expected to be covered by the javascript installation process so may be included twice.

Reply:
Indeed [we mention that they should add these in their application.js](https://github.com/alphagov/govuk_publishing_components/pull/571/files#diff-e0b066a7db963c9c680eded09ad7a928R73); so these will be imported at the application/consumer level, as we do with the scss.